bundle/EventListener/RenderViewSubscriber.php

```diff
@@ -23,10 +27,13 @@ public function __construct(ResponseTagger $responseTagger)
 
     public static function getSubscribedEvents(): array
     {
-        return [SiteApiEvents::RENDER_CONTENT => 'onRenderContent'];
+        return [
+            Events::RENDER_VIEW => 'tagResponse',
+            SiteApiEvents::RENDER_CONTENT => 'tagResponse',
+        ];
     }
 
-    public function onRenderContent(RenderContentEvent $event): void
+    public function tagResponse(RenderViewEvent $event): void
```

Comment:
Shouldn't the old `RenderContentEvent` extend `RenderViewEvent` (currently it is the opposite) to support this?

Reply:
Indeed, but I realized we shouldn't handle both dispatches in the first place, so I'll remove the the deprecated one instead.

Reply:
And maybe name the method `tagView`, since that is what you're tagging here.

Reply:
Fixed in https://github.com/netgen/ezplatform-site-api/pull/146/commits/0b248a703dba83c217794063904c3dd29592cb50

Reply:
> but I realized we shouldn't handle both dispatches in the first place

Ha, indeed, this is a built in listener after all :D

But the reversal of extending still stands, if someone would implement a listener that works on both 3.x and 4.x.

Reply:
Yup, I kept the dispatch. But we're removing this in 4.0, there will be 3.5 for deprecation :)

Reply:
I'm aware, but if it's not reversed, you can't really write a listener that will e.g. support 3.4, 3.5 and 4.0. So in `master`, `RenderContentEvent` should definitely extend `RenderViewEvent`, even in future 3.5 version, and on 4.0, just remove `RenderContentEvent`.

Reply:
Fixed extension in https://github.com/netgen/ezplatform-site-api/pull/146/commits/8bec6c2c77266db036d2da2e32fe78cf7bdbf702